src/applications/ivc-champva/10-10D/tests/config/form.unit.spec.jsx

```diff
@@ -44,6 +59,59 @@ testNumberOfWebComponentFields(
   { applicants },
 );
 
+/*
+// Commented out because this page doesn't exist currently (but may in future)
+testNumberOfWebComponentFields(
+  formConfig,
+  formConfig.chapters.applicantInformation.pages.page20.schema,
+  formConfig.chapters.applicantInformation.pages.page20.uiSchema,
+  0,
+  'Upload supporting documents',
+  { applicants },
+);
+*/
+
+testComponentRender('FileFieldCustom', <FileFieldCustom data={{}} />);
+testComponentRender(
+  'FileViewField',
+  <FileViewField
+    data={{ supportingDocuments: [{ f1: { name: 'f1', size: 123 } }] }}
+  />,
+);
+
+describe('FileFieldCustom remove button', () => {
+  it('should remove files when clicked', async () => {
+    const component = (
+      <FileFieldCustom
+        data={{ supportingDocuments: [{ name: 'filetest', size: 100 }] }}
+      />
+    );
+    const { mockStore } = getProps();
+
+    const view = render(<Provider store={mockStore}>{component}</Provider>);
+
+    const buttons = $$('va-button', $('.attachment-file', view.container));
+    expect(buttons.length === 1).to.be.true;
+    fireEvent.click(buttons[0]);
+
+    await waitFor(() => {
+      expect($('.no-attachments', view.container)).to.exist;
+    });
+  });
+});
+
+describe('File sizes', () => {
+  it('should be in bytes for values < 999', () => {
+    expect(getFileSize(998) === '998 B').to.be.true;
```

Comment:
Using `to.equal()` might increase readability.
`expect(getFileSize(998)).to.equal('998 B');`